src/metaxy/models/feature_spec.py

```diff
@@ -218,6 +241,10 @@ class _BaseFeatureSpec(pydantic.BaseModel):
             )
         ]
     )
+    metadata: MetadataField = pydantic.Field(
+        default=None,
```

Comment:
Can we *not* have this type Optional to simplify usage patterns? An empty dict is a better default (with default_factory).

Reply:
fixed to empty dict